src/common/metrics/case_density.ts

```diff
@@ -52,5 +54,27 @@ rate). Note that this will not match reported cases in many states, as
 testing is only detecting a small number of actual cases.`;
 
 export const caseDensityStatusText = (projection: Projection) => {
-  return 'Case Density Status Text';
+  const { currentCaseDensity, currentDailyDeaths, locationName } = projection;
+
+  if (currentCaseDensity === null || currentDailyDeaths === null) {
+    return `Not enough case data is available to generate ${METRIC_NAME.toLowerCase()}.`;
+  }
+
+  const level = getLevel(Metric.CASE_DENSITY, currentCaseDensity);
+  const dailyCases = formatDecimal(currentCaseDensity, 1);
+  const dailyDeaths = formatDecimal(currentDailyDeaths, 2);
+
+  const statusText1 = `Over the last week, ${locationName} has reported
+   ${dailyCases} new cases and ${dailyDeaths} new deaths per day for every
+    100,000 residents.`;
+
+  const statusText2 = levelText(
+    level,
+    `If these rates continue, we estimate less than 1% of ${locationName}’s population will be infected in the next year.`,
+    `If these rates continue, we estimate less than 1-10% of ${locationName}’s population will be infected in the next year.`,
+    `If these rates continue, we estimate less than 10-50% of ${locationName}’s population will be infected in the next year. Caution is warranted.`,
+    `If these rates continue, we estimate less than >50% of ${locationName}’s population will be infected in the next year. Aggressive action urgently needed.`,
```

Comment:
FWIW- 'less than >50%' doesn't really make sense.  

But I think all of this copy will probably need tweaking.

Reply:
good catch - I'm updating it in the PR and adding a note in the copy doc.